sql/table.sql

```diff
@@ -295,8 +295,8 @@ CREATE TABLE IF NOT EXISTS meta (
 
   meta_id                     INT NOT NULL AUTO_INCREMENT,
   species_id                  INT UNSIGNED DEFAULT 1,
-  meta_key                    VARCHAR(40) NOT NULL,
-  meta_value                  VARCHAR(255) NOT NULL,
+  meta_key                    VARCHAR(64) NOT NULL,
+  meta_value                  VARCHAR(255) NULL,
```

Comment:
Would you mind to change this for consistency to 
`meta_value                  VARCHAR(255) DEFAULT NULL,`

Thx

Reply:
Done my dear sir.